client/navigation/components/header/index.js

```diff
@@ -15,6 +16,22 @@ const Header = () => {
 		document.body.classList.toggle( 'is-folded' );
 	};
 
+	const foldOnMobile = ( screenWidth ) => {
+		if ( screenWidth <= 960 ) {
+			document.body.classList.add( 'is-folded' );
+		}
+	};
+
+	useEffect( () => {
+		foldOnMobile( document.body.clientWidth );
+
+		window.addEventListener(
+			'orientationchange',
+			( e ) => foldOnMobile( e.target.screen.availWidth ),
```

Comment:
Should this use `width` instead of `availWidth`?

Reply:
`document.body.clientWidth` and `screen.availWidth` give me differing results if I have Chrome console open. Should we stick with one or the other?